server/src/core/user/user-service.ts

```diff
@@ -379,6 +454,43 @@ export class Users implements UserService {
     }
   }
 
+  async getNew(userId: string): Promise<User> {
+    validators.validatesGetNew(userId);
+    try {
+      const user = await this.collection.findOne({ _id: userId, password: '' }, { projection: SAFE_USER_PROJECTION });
+      if (!user) throw createResourceNotFoundError(messages.ERROR_USER_NOT_FOUND);
+
+      return getSafeUser(user);
+    }
+    catch(e) {
+      rethrowIfAppError(e);
+      throw createDbOpFailedError(e.message);
+    }
+  }
```

Comment:
What does this method do?

Reply:
The method is for fetching a specific user account that has yet to be activated (i.e.,  whose password field is set to an empty string)